Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,59 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string equation)
+        {         
             // Add your code here.
-            throw new NotImplementedException();
+            //Splitting The equation into three variables eg. 42*47=1?74 to A=42 B=47 and C=1?74
+            var A = equation.Split('*')[0];
+            var B = equation.Split('*')[1].Split('=')[0];

```

Comment:
equation.Split on '*' used twice, you can maintain a variable which can store this array.
String operations are costly.

Reply:
equation.Split('*')[1].Split('=')[0] simplify this so it is readable

Reply:
string[] splitOnMultiply = equation.Split('*');
            var A = splitOnMultiply[0];
            string[] splitOnEqual = splitOnMultiply[1].Split('=');
            var B = splitOnEqual[0];
            var C = splitOnEqual[1];

sir, this is solution i come up with... 
@divastavisca

Reply:
Thats better, update your code accordingingly to resolve the review commit